src/arr/trove/matrices.arr

```diff
@@ -351,7 +352,7 @@ sharing:
   
   method to-vector(self) -> Vector:
     doc: "Returns the one-row/one-column matrix as a vector"
-    if ((self.cols == 1) <> (self.rows == 1)):
+    if ((self.cols == 1) or (self.rows == 1)):
```

Comment:
Good catch, oof!  Can you add the missing two test cases for this method to confirm that both `[row-matrix: 1, 2, 3].to-vector()` and `[col-matrix: 1, 2, 3].to-vector()` work without boolean oopses?

Reply:
Done³

Reply:
Thanks.  In hindsight, I realized what _actually_ was wrong, and the test that _actually_ was needed, would be `[row-matrix: 1]`, since we were saying "is-row-matrix isn't equal to is-col-matrix", which would be incorrect when it's a 1x1 matrix...but no need to add that test for now; I'm satisfied this is obviously less dumb than the code I'd checked in :)